vars/githubscm.groovy

```diff
@@ -92,6 +92,7 @@ def commitChanges(String userName, String userEmail, String commitMessage, Strin
 }
 
 def forkRepo(String credentialID='kie-ci') {
+    sh "rm -rf ~/.config/hub"
```

Comment:
Coudl we extract that method as `cleanHubAuth()` ?

Reply:
This is just a single liner, and we want this in these functions to make sure the creds are picked from ENV var. I think it would be better to just leave it as it is.

Reply:
On second thoughts let me extract it as a function.